types/collectibles.go

```diff
@@ -53,6 +53,14 @@ type (
 		TransferFee       *CollectibleTransferFee `json:"transfer_fee,omitempty"`
 		PreviewImageURL   *CollectibleMedia       `json:"preview_image_url,omitempty"`
 		OriginalSourceURL CollectibleMedia        `json:"original_source_url"`
+		Properties        []CollectibleProperty   `json:"properties"`
+		About             string                  `json:"about"`
+	}
+
+	CollectibleProperty struct {
+		Key    string  `json:"key"`
+		Value  string  `json:"value"`
+		Rarity float32 `json:"rarity,omitempty"`
```

Comment:
Can Rarity be zero?
If yes then omitempty will exclude it from json, otherwise it's ok.

Reply:
Yes, it can be zero and will be excluded from response